### PR TITLE
Sphinx theme skeleton

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds a skeleton for a pip-installable theme that I worked on during the sprints at PyCon 2015. Highlights:
- Added an MIT license file
- Switched the README to reStructuredText to support eventual inclusion on PyPI
- Substantiated the README with installation instructions
- Added a `pyladies_sphinx_theme` package (I think there was a mention of a PyLadies namespace, so I imagine this will change along with other metadata)

I'm open to feedback or changes to make this better. Thanks!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.. _Read the Docs: https://readthedocs.org/
+
+
+How to use this theme
+---------------------
+
+To use this theme:
+
+#. Install the theme.
+   Run ``pip install pyladies_sphinx_theme``.
```
Comment (econchick) [inline README.rst]: I think with pip, it is (or should be) `pyladies-sphinx-theme` then gets translated into underscores for import/use.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    install_requires=['sphinx'],
+
+    package_data={'pyladies_sphinx_theme': [
+        'theme.conf',
+        '*.html',
+        'static/css/*.css',
+    ]},
+
+    zip_safe=False,
+)
```
Comment (econchick) [inline setup.py:42]: I'd suggest to remove the extraneous spaces for each line.

Comment (econchick) [inline setup.py:42]: (just for the `setup()` call)

File: pyladies_sphinx_theme/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import os.path
```
Comment (econchick) [inline pyladies_sphinx_theme/__init__.py:4]: Would adding the encoding line here be okay?  And the Copyright? e.g.:

``` python
# -*- coding: utf-8 -*-
# Copyright (c) 2015 PyLadies
```

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import codecs
```
Comment (econchick) [inline setup.py:4]: Same question as above re encoding & copyright

**Comment by econchick** [comment]
Hey @ddbeck - sorry for the super long response.  Thank you again so much for doing this!  Let me know what you think w/r/t the comments I gave.

**Comment by ddbeck** [comment]
Thanks for taking the time to review everything, @econchick. My initial impression is :+1: for everything, but I will give everything a closer look, test all the changes, and update the PR in the next 72 hours.

**Comment by econchick** [comment]
No rush :)

**Comment by ddbeck** [comment]
@econchick OK, I made all the changes you suggested, though I changed the underscores to dashes in the actual package name _and_ the readme with 75dbae0. This way the package name on PyPI and in the install command will be the same (`pyladies-sphinx-theme`).

**Comment by econchick** [comment]
Wow I can't believe I missed this - I'm so sorry @ddbeck !  

This looks great - I'm merging!

**Pull request merged**